client/src/search/Typeahead/Typeahead.js

```diff
@@ -0,0 +1,217 @@
+          menu_content: (search) =>
+            _.isFunction(search_config.menu_content_function) ? (
+              search_config.menu_content_function(data, search)
+            ) : (
+              <InfoBaseHighlighter
+                search={search}
+                content={search_config.name_function(data)}
+              />
+            ),
+          config_group_index: ix,
+        }))
+      ),
+    ];
+
+    const matches_query = (option) => {
+      const { search_text } = this.state;
+      const group_filter =
+        config_groups[option.config_group_index].group_filter;
+      const query_matches = group_filter(search_text, option.data);
+
+      return query_matches;
+    };
+
+    const queried_results = _.filter(
+      all_options,
+      (res) => matches_query(res) && !_.isUndefined(res.config_group_index)
+    );
+
+    const menu_props = {
+      queried_results,
+      pagination_size,
+      search_text,
+      config_groups,
+      on_select_item: this.on_select_item,
+      hide_menu: this.hide_menu,
+    };
+
+    return (
+      <div className="rbt" style={{ position: "relative" }}>
+        <div className="search-bar">
+          <div className="search-icon-container">
+            <span aria-hidden="true">
+              <img
+                src={`${get_static_url("svg/search.svg")}`}
+                style={{ width: "30px", height: "30px" }}
+              />
+            </span>
+          </div>
+          <input
+            style={{ width: "100%" }}
+            placeholder={placeholder}
+            aria-lable={placeholder}
+            onChange={update_search_text}
+            ref={this.typeaheadRef}
+            onKeyDown={this.handleKeyDown}
+            value={search_text}
+            onFocus={() => this.setState({ can_show_menu: true })}
+            onKeyDown={this.handle_key_down}
+          />
+          {filter_content && (
+            <DropdownMenu
+              dropdown_trigger_txt={
+                <div
+                  style={{
+                    textAlign: "start",
+                    whiteSpace: "nowrap",
+                    display: "inline-block",
+                  }}
+                >
+                  <MediaQuery minWidth={breakpoints.minSmallDevice}>
+                    <div
+                      style={{
+                        whiteSpace: "nowrap",
+                        display: "inline-block",
+                        marginRight: "1.5rem",
+                      }}
+                    >
+                      <IconFilter
+                        height="5px"
+                        width="5px"
+                        vertical_align="top"
+                        alternate_color="false"
+                      />
+                    </div>
+                  </MediaQuery>
+                  <span>{text_maker("filter")}</span>
+                </div>
+              }
+              dropdown_close_txt={text_maker("filter")}
```

Comment:
Just noticed, this prop doesn't exist. Can you make sure all other Dropdowns are consistent as well?